### DISCUSSION TITLE
Displaying current line number, and ideally a 'go to line #' feature.

### DISCUSSION THREAD

**Discussion by silentdiverchris** [KDE/ghostwriter#656] [Ideas]
Hi there, am a recent but keen fan of your creation and have a small enhancement request if I may.

I am writing a [to-do system](https://github.com/silentdiverchris/Jiminy) that is based on adding custom tags to Markdown files which then get extracted, correlated and displayed elsewhere. 

The display refers to the source files with the line number where it found the tags, so a frequent operation for me is to open up a Markdown file in Ghostwriter and look for that line.

It would be very handy if the current line number, where the insertion point is, was displayed. Next to the word count would be a good spot, to make it easier to navigate to the line in question. 

Even better, if there was also a simple 'go to line number' feature hanging off CTRL-G as per Visual Studio and Notepad++ that would be great.

...and thinking to the future, it would be even much betterer if one could supply a command parameter that made it go to that line on opening a file. I'm assuming you can supply a file name for it to open already (edit: I checked, it does, great). The display my package creates is HTML so it can't auto-open the source file but I'll probably write a native front end for it at some point and it'd be good to have it be able to to open the source at the right spot in Ghostwriter.

Cheers again for the work putting Ghostwriter together, I've always used Notepad++ to generate .md content and while it's a fantastic text editor of course, I recently decided I wanted something more purpose-built so looked around at the alternatives and found yours to be my favourite, very nice, thanks.

Cheers, Chris

**Comment by Jivix-zz** [comment]
Some of these would be relevant to how I am using Ghostwriter as well - would love to see the app be able to do this.
